docs/platforms/unreal/tracing/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Set Up Tracing
+description: "Learn how to enable tracing in your app and discover valuable performance insights of your application."
+sidebar_order: 4000
+---
+
+With [tracing](/product/performance/), Sentry tracks your software performance, measuring metrics like throughput and latency, and displaying the impact of errors across multiple systems. Sentry captures distributed traces consisting of transactions and spans, which measure individual services and individual operations within those services. Learn more about our model in [Distributed Tracing](/product/sentry-basics/tracing/distributed-tracing/).
+
+<Note>
+
+If you’re adopting Tracing in a high-throughput environment, we recommend testing prior to deployment to ensure that your service’s performance characteristics maintain expectations.
+
+</Note>
+
+## Configure
+
+First, enable tracing and configure the sample rate for transactions. Set the sample rate for your transactions by either:
+
+- Setting a uniform sample rate for all transactions using the <PlatformIdentifier name="traces-sample-rate" /> option in your SDK config to a number between `0` and `1`. (For example, to send 20% of transactions, set <PlatformIdentifier name="traces-sample-rate" /> to `0.2`.)
+- Controlling the sample rate based on the transaction itself and the context in which it's captured, by providing a function to the <PlatformIdentifier name="traces-sampler" /> config option.
+
+The two options are meant to be mutually exclusive. If you set both, <PlatformIdentifier name="traces-sampler" /> will take precedence.
+
+<Alert level="info">
+
+The Unreal Engine SDK doesn't currently support sampling functions on Windows/Linux (<PlatformIdentifier name="traces-sampler" />).
```

Comment:
This note is duplicated, do we need it twice?
```suggestion
Currently there is no support for sampling functions on Windows/Linux (<PlatformIdentifier name="traces-sampler" />).
```

Reply:
I think it makes sense to mention this on both `Basic Options` and `Set Up Tracing` pages to avoid potential confusion about sampling functions limited platform support.